mbq/atomiq/management/commands/atomic_run_consumer.py

```diff
@@ -29,23 +31,66 @@ def __init__(self, *args, **kwargs):
         signal.signal(signal.SIGTERM, self.signal_handler.handle_signal)
         signal.signal(signal.SIGQUIT, self.signal_handler.handle_signal)
 
+        self.queues = {
+            constants.QueueType.SNS: {
+                'model': models.SNSTask,
+                'consumer_class': consumers.SNSConsumer,
+            },
+            constants.QueueType.SQS: {
+                'model': models.SQSTask,
+                'consumer_class': consumers.SQSConsumer,
+            },
+            constants.QueueType.CELERY: {
+                'model': models.CeleryTask,
+                'consumer_class': consumers.CeleryConsumer,
+            },
+        }
+
```

Comment:
nit: The consumers already hold a reference to the Queue model they use, so you can avoid duplicating the association of the model with the consumer like so: `SNSConsumer.model`, `SQSConsumer.model`, `CeleryConsumer.model`.